From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sun, 20 Mar 2016 18:24:04 +0100
Subject: [PATCH] m1.micro: Increase profile RAM size to 128MB.

TestVM (cirros) on aarch64 requires more than 64MB RAM.
Keep profiles uniform across all archs.
---
 .../osnailyfacter/modular/openstack-controller/openstack-controller.pp  | 2 +-
 tests/noop/spec/hosts/openstack-controller/openstack-controller_spec.rb | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/deployment/puppet/osnailyfacter/modular/openstack-controller/openstack-controller.pp b/deployment/puppet/osnailyfacter/modular/openstack-controller/openstack-controller.pp
index b1d8795..88e0ab3 100644
--- a/deployment/puppet/osnailyfacter/modular/openstack-controller/openstack-controller.pp
+++ b/deployment/puppet/osnailyfacter/modular/openstack-controller/openstack-controller.pp
@@ -233,7 +233,7 @@ if $primary_controller {
       "OS_REGION_NAME=${region}",
       "NOVA_ENDPOINT_TYPE=internalURL",
     ],
-    command   => 'bash -c "nova flavor-create --is-public true m1.micro auto 64 0 1"',
+    command   => 'bash -c "nova flavor-create --is-public true m1.micro auto 128 0 1"',
     #FIXME(mattymo): Upstream bug PUP-2299 for retries in unless/onlyif
     # Retry nova-flavor list until it exits 0, then exit with grep status,
     # finally exit 1 if tries exceeded
diff --git a/tests/noop/spec/hosts/openstack-controller/openstack-controller_spec.rb b/tests/noop/spec/hosts/openstack-controller/openstack-controller_spec.rb
index 36161e6..b15da07 100644
--- a/tests/noop/spec/hosts/openstack-controller/openstack-controller_spec.rb
+++ b/tests/noop/spec/hosts/openstack-controller/openstack-controller_spec.rb
@@ -171,7 +171,7 @@ describe manifest do
     if primary_controller
       it 'should retry unless when creating m1.micro flavor' do
         should contain_exec('create-m1.micro-flavor').with(
-           'command' => 'bash -c "nova flavor-create --is-public true m1.micro auto 64 0 1"',
+           'command' => 'bash -c "nova flavor-create --is-public true m1.micro auto 128 0 1"',
            'unless'  => 'bash -c \'for tries in {1..10}; do
                     nova flavor-list | grep m1.micro;
                     status=("${PIPESTATUS[@]}");
